### PR TITLE
fix(28081): design tweak for network badge

### LINKED ISSUE #28081: [Design quality]: Update border weight and color for networks and status badges
### Describe the bug

There is inconsistent border width for badges (status and networks) which makes the Portfolio View spacing feel off. Updating this improves overall visual cohesion and product quality. 

### Expected behavior

Multichain menu badge (connected sites)
- `border width` = `3px` > `2px`

Network badge updates
- `border width` = `1px` > `2px`
- `color` = `bg.alternative` > `bg.default`
- Make sure changes are reflected in the `Asset Picker` and `Import Tokens`

Network menu
- Remove border in default, selected and hover states

Account badge
- `border width` = `4px` > `2px`

### Screenshots/Recordings

Screenshot 2024-10-25 at 12 18 29 AM

[Figma link](https://www.figma.com/design/aMYisczaJyEsYl1TYdcPUL/Portfolio-View?node-id=6235-15524&t=x7GxHjVI19oISUoI-4)

Screenshot 2024-10-25 at 12 26 09 AM

[Figma link](https://www.figma.com/design/aMYisczaJyEsYl1TYdcPUL/Portfolio-View?node-id=6235-15566&t=x7GxHjVI19oISUoI-4)

Screenshot 2024-10-25 at 12 30 11 AM

[Figma link](https://www.figma.com/design/aMYisczaJyEsYl1TYdcPUL/Portfolio-View?node-id=6235-15580&t=x7GxHjVI19oISUoI-4)

### Steps to reproduce

1. Open MetaMask
2. Open network menu
3. Connect account
4. Click on account selector

### Error messages or log output

_No response_

### Detection stage

In production (default)

### Version

12.5.0

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

Ping @amandaye0h and @georgewrmarshall for reviews

### Severity

_No response_

### PR DESCRIPTION
**Pull request by DDDDDanica**
## **Description**
This PR address feedback from design quality check to tweak board color and width for network badge.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/29324?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/28081

## **Manual testing steps**

1. Go to this page...
2.
3.

### **Before**
See PR for related changes

### **After**
See PR for related changes

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/components/multichain/connected-site-menu/connected-site-menu.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const ConnectedSiteMenu = ({
                   ? BorderColor.successDefault
                   : BackgroundColor.backgroundDefault
               }
-              borderWidth={isConnectedtoOtherAccountOrSnap ? 2 : 3}
+              borderWidth={2}
```
Comment (DDDDDanica) [inline ui/components/multichain/connected-site-menu/connected-site-menu.js:93]: Before: 
Screenshot 2024-12-18 at 15 29 05

After: 
Screenshot 2024-12-18 at 15 08 14

File: ui/components/multichain/token-list-item/token-list-item.tsx [hunk 2]
```diff
@@ -285,7 +283,8 @@ export const TokenListItem = ({
               size={AvatarNetworkSize.Xs}
               name={allNetworks?.[chainId as Hex]?.name}
               src={tokenChainImage || undefined}
-              backgroundColor={testNetworkBackgroundColor}
+              backgroundColor={BackgroundColor.backgroundDefault}
+              borderWidth={2}
```
Comment (DDDDDanica) [inline ui/components/multichain/token-list-item/token-list-item.tsx:287]: Before: 
Screenshot 2024-12-18 at 13 40 01

After: 
Screenshot 2024-12-18 at 13 11 18

CSS breakdown: 
Screenshot 2024-12-18 at 13 20 28

File: ui/components/multichain/badge-status/badge-status.tsx [hunk 3]
```diff
@@ -74,7 +74,7 @@ export const BadgeStatus: React.FC<BadgeStatusProps> = ({
               backgroundColor={badgeBackgroundColor}
               borderRadius={BorderRadius.full}
               borderColor={badgeBorderColor}
-              borderWidth={isConnectedAndNotActive ? 2 : 4}
+              borderWidth={2}
```
Comment (DDDDDanica) [inline ui/components/multichain/badge-status/badge-status.tsx:77]: Before:
Screenshot 2024-12-18 at 15 29 05

After: 
Screenshot 2024-12-18 at 15 08 14

File: ui/components/multichain/connected-site-menu/connected-site-menu.js (hunk 1 above)
Comment (DDDDDanica) [inline ui/components/multichain/connected-site-menu/connected-site-menu.js:93]: Before (not connected):
Screenshot 2024-12-18 at 15 41 03

After (not connected):
Screenshot 2024-12-18 at 15 39 32

Before (connected):
Screenshot 2024-12-18 at 15 41 21

After (connected)
Screenshot 2024-12-18 at 15 39 17
:

File: ui/components/multichain/account-list-item/account-list-item.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ import {
   getMultichainShouldShowFiat,
 } from '../../../selectors/multichain';
 import { useMultichainAccountTotalFiatBalance } from '../../../hooks/useMultichainAccountTotalFiatBalance';
-import { ConnectedStatus } from '../connected-status/connected-status';
+import { ConnectedStatus } from '../connected-status';
```
Comment (DDDDDanica) [inline ui/components/multichain/account-list-item/account-list-item.js:65]: Path simplification

File: ui/components/multichain/badge-status/badge-status.stories.tsx [hunk 5]
```diff
@@ -42,7 +42,7 @@ export const DefaultStory = Template.bind({});
 
 export const NotConnectedStory = Template.bind({});
 NotConnectedStory.args = {
-  badgeBackgroundColor: Color.borderMuted,
+  badgeBackgroundColor: BackgroundColor.iconAlternative,
```
Comment (georgewrmarshall) [inline ui/components/multichain/badge-status/badge-status.stories.tsx:45]: ### Before

TBC

### After 

Screenshot 2024-12-19 at 7 40 20 AM

File: ui/components/multichain/badge-status/badge-status.tsx (hunk 3 above)
Comment (georgewrmarshall) [reply ui/components/multichain/badge-status/badge-status.tsx:77]: Updated so now the background color of the badge matches the header icon

### After

Screenshot 2024-12-19 at 7 41 54 AM
Screenshot 2024-12-19 at 7 42 08 AM

File: ui/components/multichain/connected-status/connected-status.tsx [hunk 6]
```diff
@@ -43,11 +43,11 @@ export const ConnectedStatus: React.FC<ConnectedStatusProps> = ({
     status = STATUS_CONNECTED_TO_ANOTHER_ACCOUNT;
   }
 
-  let badgeBorderColor = BackgroundColor.backgroundDefault; // TODO: Replace it once border-color has this value.
-  let badgeBackgroundColor = Color.borderMuted; // //TODO: Replace it once Background color has this value.
+  let badgeBorderColor = BorderColor.backgroundDefault; // TODO: Replace it once border-color has this value.
+  let badgeBackgroundColor = BackgroundColor.iconAlternative;
```
Comment (georgewrmarshall) [inline ui/components/multichain/connected-status/connected-status.tsx:47]: We recently updated border muted to be transparent which is why the UI looked "broken" I've updated it to use icon/alternative. This also matches Figma

Screenshot 2024-12-19 at 7 41 54 AM
Screenshot 2024-12-19 at 7 42 08 AM
Screenshot 2024-12-19 at 7 43 48 AM

File: ui/components/app/detected-token/detected-token-details/detected-token-details.js [hunk 7]
```diff
@@ -43,6 +43,7 @@ const DetectedTokenDetails = ({
             src={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
             name={currentNetwork?.nickname || ''}
             backgroundColor={testNetworkBackgroundColor}
+            borderWidth={2}
```
Comment (georgewrmarshall) [inline ui/components/app/detected-token/detected-token-details/detected-token-details.js:46]: ### After

Screenshot 2024-12-19 at 7 48 02 AM

File: ui/components/multichain/token-list-item/token-list-item.tsx (hunk 2 above)
Comment (georgewrmarshall) [reply ui/components/multichain/token-list-item/token-list-item.tsx:287]: Hey @DDDDDanica, should the before and after images be swapped? For example, should the 2px border be *after* and the 1px border be *before*?

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! 🔥  I made some small updates hope that is ok!

File: ui/components/multichain/token-list-item/token-list-item.tsx (hunk 2 above)
Comment (DDDDDanica) [reply ui/components/multichain/token-list-item/token-list-item.tsx:287]: Good spot ! switched 🙏🏻

File: ui/components/app/snaps/snap-ui-address/__snapshots__/snap-ui-address.test.tsx.snap [hunk 8]
```diff
@@ -476,7 +476,7 @@ exports[`SnapUIAddress renders legacy Ethereum address 1`] = `
           <rect
             fill="#F29602"
             height="32"
-            transform="translate(-2.81750896228304 4.951916544006229) rotate(230.4 16 16)"
+            transform="translate(-2.8175089622830405 4.951916544006229) rotate(230.4 16 16)"
             width="32"
             x="0"
             y="0"
```
[the remark this answers is not in the document]
Comment (DDDDDanica) [reply ui/components/app/snaps/snap-ui-address/__snapshots__/snap-ui-address.test.tsx.snap:482]: I found similar thing in weird svg snapshot failure, FAIL ui/components/multichain/connect-accounts-modal/connect-accounts-modal.test.tsx (5.461 s), and this one only failed in github actions

[the remark this answers is not in the document]
Comment (DDDDDanica) [reply ui/components/app/snaps/snap-ui-address/__snapshots__/snap-ui-address.test.tsx.snap]: Trying to manually applying those snapshot changes locally and see how it goes 🙇🏻‍♀️

[the remark this answers is not in the document]
Comment (DDDDDanica) [reply ui/components/app/snaps/snap-ui-address/__snapshots__/snap-ui-address.test.tsx.snap]: Addressed: c27d151f4f9424874ef68ec288827e6657b0090a

**Review by pedronfigueiredo — APPROVED** [review]

**Review by dbrans — APPROVED** [review]
Nice work!

**Review by NidhiKJha — APPROVED** [review]
LGTM

**Pull request merged**